srcpkgs/jbig2enc/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'jbig2enc'
+pkgname=jbig2enc
+version=0.29
+revision=1
+#archs="i686 x86_64"
+#build_wrksrc=
+build_style=gnu-configure
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="autoconf automake pkg-config libtool"
+makedepends="leptonica-devel tiff-devel libpng-devel giflib-devel libwebp-devel"
+depends="leptonica tiff libpng giflib libwebp python3"
```

Comment:
Most of those will be detected automatically, you can leave it at just `depends="python3"`

Reply:
Neat! Thanks for the tip.